src/bundle/Resources/public/js/scripts/fieldType/ezbinaryfile.js

```diff
@@ -10,9 +9,9 @@
          * @param {Event} event
          */
         loadDroppedFilePreview(event) {
-            const preview = this.fieldContainer.querySelector('.ez-field-edit__preview');
-            const nameContainer = preview.querySelector('.ez-field-edit-preview__file-name');
-            const sizeContainer = preview.querySelector('.ez-field-edit-preview__file-size');
+            const preview = this.fieldContainer.querySelector('.ibexa-field-edit__preview');
+            const nameContainer = preview.querySelector('.ibexa-field-edit-preview__file-name p');
```

Comment:
I think we should add `class` tag to `p` element - maybe replace `__file-name` with `__file-name-wrapper` and add `__file-name` to `p` element?